tdrs-backend/tdpservice/core/views.py

```diff
@@ -0,0 +1,18 @@
+"""Define core, generic views of the app."""
+import logging
+
+from rest_framework.response import Response
+from rest_framework.decorators import api_view
+
+logger = logging.getLogger()
+
+
+@api_view(['POST'])
+def write_logs(request):
+    """Pass request bodies to the system logger.
+
+    Mainly used to log client-side alerts and errors.
+    """
+    logger.info(request.data)
+
```

Comment:
Instead of just printing the dictionary as is, what if we were a little more deliberate?
```python
for key in request.data:
    logger.info(f"{key}: {request.data[key]}")
```
I think it would make the logs easier to read. Thoughts?

Reply:
Agreed, I asked about this earlier and it got lost in the noise.